go/cli/mcap/cmd/merge.go

```diff
@@ -361,4 +409,11 @@ func init() {
 		true,
 		"chunk the output file",
 	)
+	mergeCmd.PersistentFlags().StringVarP(
+		&coalesceChannels,
+		"coalesce-channels",
+		"",
+		"auto",
+		"channel coalescing behavior (supported: auto, force, none)",
```

Comment:
Can we explain the behavior of each further in the help command output?

Reply:
Done. Shows up like this now:
![image](https://github.com/foxglove/mcap/assets/9250155/5566efe1-9918-49f1-b2f6-83b0566315fc)